packages/kotti-ui/source/kotti-field-file-upload/components/DropArea.vue

```diff
@@ -0,0 +1,276 @@
+<template>
+	<div
+		ref="dropAreaRef"
+		:class="classes"
+		:data-test="dataTest ? `${dataTest}.dropArea` : undefined"
+		:tabIndex="_tabIndex"
+		@click.prevent="onClick"
+		@dragleave.prevent.stop="isDragging = false"
+		@dragover.prevent.stop="isDragging = true"
+		@drop.prevent.stop="onDrop"
+		@mouseenter.prevent.stop="isHovering = true"
+		@mouseleave.prevent.stop="isHovering = false"
+	>
+		<div class="kt-field-file-upload__drop-area__description">
+			<i class="yoco" v-text="_icon" />
+			<a v-text="translations.text.clickToUpload" />
+			{{ translations.text.dragAndDrop }}
+		</div>
+		<div
+			v-if="showInformation"
+			class="kt-field-file-upload__drop-area__information"
+		>
+			{{ informationText }}
+			<a
+				v-if="infoUrl"
+				:href="infoUrl"
+				rel="noopener noreferrer"
+				target="_blank"
+				@click.stop
+				v-text="translations.text.learnMore"
+			/>
+		</div>
+		<input
+			v-show="false"
+			ref="uploadInputRef"
+			:accept="accept"
+			:data-test="dataTest ? `${dataTest}.input` : undefined"
+			:multiple="allowMultiple"
+			type="file"
+			@change="onSelect"
+			@click.stop
+		/>
+		<div
+			v-if="$slots.content"
+			class="kt-field-file-upload__drop-area__content-slot"
+			@click.stop="onContentSlotClick"
+		>
+			<slot name="content" />
```

Comment:
👏🏼

Reply:
Thinking out loud:

do we need to expose anything from this slot to the user-app

i understand why you're using a slot on this level if you detect one on the parent and you have the handler to guard against "disabled" clicks

but if the user-app uses @click.stop on the slot--> it won't bubble up the click event anyways

so what I'm thinking is: "is it better design to just expose the isDisabled prop?"